yarn-project/simulator/src/avm/serialization/bytecode_serialization.ts

```diff
@@ -2,7 +2,6 @@ import { PedersenCommitment } from '../opcodes/commitment.js';
 import { DAGasLeft, L2GasLeft } from '../opcodes/context_getters.js';
 import { EcAdd } from '../opcodes/ec_add.js';
 import { Keccak, KeccakF1600, Pedersen, Poseidon2, Sha256 } from '../opcodes/hashing.js';
```

Comment:
What is the reason that we don't have them all imported from the index?

Reply:
Might have been just VSCode autocompleting